### PR TITLE
start refactor of update for pip installs

### PR DESCRIPTION
**Pull request by evansde77**
@shudgston 

The venv lookup in build seemed broken, tweaked it to use virtualenv_home() instead

### CODE REVIEW

**Comment by shudgston** [comment]
Worked for me +1

File: src/cirrus/build.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
 import sys
 from argparse import ArgumentParser
 
-from cirrus.environment import cirrus_home
+from cirrus.environment import cirrus_home, virtualenv_home
```
Comment (shudgston) [inline src/cirrus/build.py]: Unused?

File: src/cirrus/selfupdate.py [hunk 2]
```diff
@@ -94,6 +105,9 @@ def latest_release(config):
     most_recent_tag = tags[0][0]
     return most_recent_tag
 
+def latest_pypi_release():
+    curl https://pypi.python.org/pypi/cirrus-cli/json | jq .info.release_url
```
Comment (shudgston) [inline src/cirrus/selfupdate.py]: What?

File: src/cirrus/build.py (hunk 1 above)
Comment (evansde77) [inline src/cirrus/build.py]: yup!

**Pull request merged**